test/e2e/run_test.go

```diff
@@ -491,12 +491,18 @@ var _ = Describe("Podman run", func() {
 		session.WaitWithDefaultTimeout()
 		Expect(session.ExitCode()).To(Equal(0))
```

Comment:
There's some lines above this that disable the test on Travis - can you strip those out so we can verify the timeout is gone?

Reply:
Done